pusher/src/Controller/IoSocketController.ts

```diff
@@ -281,7 +281,7 @@ export class IoSocketController {
                         let memberMessages: unknown;
                         let memberUserRoomToken: string | undefined;
                         let memberTextures: WokaDetail[] = [];
-                        const room = await socketManager.getOrCreateRoom(roomId);
+                        const room = await socketManager.getOrCreateRoom(roomId, userIdentifier);
```

Comment:
Here, the parameters makes no sense.

If I want to getOrCreate a room, I need a roomId. Passing a user of the room should definitely not be required.